packages/collector/test/tracing/frameworks/express/app.js

```diff
@@ -13,6 +13,14 @@ process.on('SIGTERM', () => {
   process.exit(0);
 });
 
+const mock = require('mock-require');
```

Comment:
suggestion: use the same approach as we use for the other apps. Add a `mockVersion` helper file.